.github/workflows/verify_api.yml

```diff
@@ -0,0 +1,46 @@
+name: Verify API
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'scripts/*'
+      - 'api-generator/*'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'scripts/**'
+      - 'api-generator/**'
+jobs:
+  verify:
+    timeout-minutes: 30
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Cache Downloaded Drivers
+        uses: actions/cache@v2
+        with:
+          path: driver-bundle/src/main/resources/driver
+          key: ${{ runner.os }}-drivers-${{ hashFiles('scripts/*') }}
+          restore-keys: ${{ runner.os }}-drivers
+      - name: Download drivers
+        shell: bash
```

Comment:
`shell: bash` is the default shell, I would remove it on all the steps since it probably won't be changed by the GHA team and is by that redundant.

Reply:
I remember it biting us on Windows, where it is not the default

Reply:
Removing for now.

Reply:
True 👍  I thought the verify workflow is enough to run on Linux only.